vocs.config.ts

```diff
@@ -11,6 +11,14 @@ export default defineConfig({
   },
   vite: {
     publicDir: "docs/public",
+    plugins: [
+      {
+        name: "add-custom-css",
+        transformIndexHtml(html) {
+          return html.replace("</head>", '<link rel="stylesheet" href="/custom.css" /></head>');
+        },
+      },
+    ],
```

Comment:
Umm not sure if it's worth it, I think just the logo goods decent enough. 

We can also ask Andread to have the full logo as text 

<img width="349" height="61" alt="Image" src="https://github.com/user-attachments/assets/0f03cea7-fcc6-4947-88a8-e80bcda9dda8" />

Reply:
Yup yup both looks good to me, and agree that its a bit empty...just cc @Pabl0cks  once as well for his opinion